src/pages/ScholarshipForm.jsx

```diff
@@ -40,8 +40,8 @@ function ScholarshipForm() {
   });
 
   function updateAmount(minAmount, maxAmount) {
-    formik.setFieldValue('amount.minAmount', minAmount, false);
-    formik.setFieldValue('amount.maxAmount', maxAmount, false);
+    formik.setFieldValue('amount.minAmount', minAmount, true);
```

Comment:
I think `true` is the default behavior is [`validateOnChange`](https://formik.org/docs/api/formik#validateonchange-boolean) is `true` (which it is by default)